cfpb/consumerfinance.gov
# P4C: grad path tool PLUS and private loan help text fixes

Fix PLUS and private loan help text in grad path tool. See CFPT 372 for more info.


## Changes

- Help text updates on Make a Plan screen 


## How to test this PR

1. Check out branch
2. Navigate to [grad path tool](http://localhost:8000/paying-for-college/your-financial-path-to-graduation/?iped=110635&pid=4510-3&houp=onCampus&typp=bachelors&lenp=4&ratp=inState&depp=dependent&houx=1430&fdx=477&clhx=120&trnx=576&hltx=295&entx=181&retx=256&taxx=211&othx=327)
3. Verify help text changes under PLUS and private loan expandables


## Screenshots

<img width="609" alt="Screen Shot 2021-03-04 at 11 52 03 AM" src="https://user-images.githubusercontent.com/778171/110025032-b6ba1180-7ce3-11eb-8ff4-7541f1a424a5.png">
<img width="604" alt="Screen Shot 2021-03-04 at 11 52 51 AM" src="https://user-images.githubusercontent.com/778171/110025042-b9b50200-7ce3-11eb-8bfa-da36a0235686.png">
<img width="590" alt="Screen Shot 2021-03-04 at 12 00 29 PM" src="https://user-images.githubusercontent.com/778171/110025043-ba4d9880-7ce3-11eb-8fbd-0188e632af9d.png">
